cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -986,10 +986,20 @@ fn aarch64_get_operands<F: Fn(VReg) -> VReg>(inst: &Inst, collector: &mut Operan
                 collector.reg_use(ret);
             }
         }
-        &Inst::AuthenticatedRet { ref rets, .. } => {
+        &Inst::AuthenticatedRet {
+            ref rets, is_hint, ..
+        } => {
+            collector.reg_use(regs::stack_reg());
```

Comment:
This use of SP and the mod/use of LR below shouldn't be necessary, as the registers are not allocatable so the RA doesn't need to track liveranges over them. I think this match arm can work the same as the `Ret` case above, unless I'm missing some new constraint!